_scripts/collect_contributor_data.r

```diff
@@ -0,0 +1,39 @@
+
+if (!require(gh)) {
+    install.packages("gh")
+}
+if (!require(allcontributors)) {
+    install.packages("allcontributors")
+}
+
+library(allcontributors)
+library(gh)
+
+org_name <- "epiverse-trace"
+repos <- gh("GET /orgs/:org/repos", org = org_name, .limit = 100)
+# A vector of repo names in the organization that we do not include
+# This is because they are forks from external orgs, for example.
+excluded_repos <- c(
+    "sandpaper",
+    "testpkg",
+    "varnish"
+)
+
+repos <- lapply(repos, function(x) {
+    if (sum(x$name == excluded_repos) == 0) {
```

Comment:
The following is much more idiomatic in R

```suggestion
    if (!x$name %in% excluded_repos)) {
```